onnxscript/rewriter/ort_fusions/fused_matmul_rule_sets_test.py

```diff
@@ -358,6 +385,357 @@ def test_should_not_match(self):
             )
             self._check_model(model_proto, rewritten_model, atol=1e-6)
 
+    # Add unit tests to check that fusion rewrite can work even if MatMul is not the first node.
+    @classmethod
+    def _fused_matmul_with_transpose_or_matmul_in_middle(cls):
```

Comment:
Also if we are testing multiple models, consider using `parameterized` to create parameterized tests. I think we can avoid the classmethod.